npc/re/merchants/shops.txt

```diff
@@ -91,6 +91,25 @@ malangdo,232,163,5	shop	Tool Dealer Myo	560,611:-1,1750:-1,645:-1,656:-1,601:-1,
 mora,100,118,4	shop	Tool Dealer#mora	517,11522:-1,11523:-1,11524:-1,602:-1,601:-1,611:-1,1773:-1,1774:-1
 mora,138,110,4	shop	Mora Chef	518,11519:-1
 
+//=======================================================
+// Morroc
+//=======================================================
+morocc,156,76,4	shop	Butcher#moc1	58,517:-1
+moc_ruins,91,128,4	shop	Tool Dealer#moc1	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,645:-1,656:-1,713:-1,601:-1,602:-1,1065:-1
+moc_ruins,114,63,6	shop	Tool Dealer#moc2	99,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1,2242:-1
+morocc,146,103,6	shop	Tool Dealer#moc3	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1
+morocc,151,241,4	shop	Tool Dealer#moc4	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1
+moc_ruins,118,170,4	shop	Pet Groomer#moc	124,537:-1,643:-1,10013:-1,10014:-1,554:-1,6114:-1,6113:-1,6110:-1,6115:-1,6100:-1,6098:-1,6112:-1,6104:-1,6108:-1,6111:-1,6095:-1,6099:-1,6096:-1,6097:-1,6094:-1
+morocc,203,87,4	shop	Pet Groomer#moc1	124,537:-1,643:-1,10013:-1,10014:-1,554:-1,6114:-1,6113:-1,6110:-1,6115:-1,6100:-1,6098:-1,6112:-1,6104:-1,6108:-1,6111:-1,6095:-1,6099:-1,6096:-1,6097:-1,6094:-1
```

Comment:
```
trader "morocc" "테이밍상인" 4_M_TELEPORTER 203 87 4 0 0 // 52313
	sellitem 25377 // 10000
	sellitem Pet_Food // 1000
	sellitem Pet_Incubator // 10000
	sellitem Backpack // 1500
	sellitem Rocker_Glasses // 2000
	sellitem Vital_Flower_ // 1000
	sellitem Flame_Gemstone // 1000
	sellitem Bun_ // 1000
	sellitem Vital_Flower // 1000
	sellitem Damp_Darkness // 1000
	sellitem Small_Snow_Flower // 1000
	sellitem Fresh_Plant // 1000
	sellitem Big_Cell // 1000
	sellitem Apple_Pudding // 1000
	sellitem Mystic_Stone // 1000
	sellitem Flavored_Alcohol // 1000
	sellitem Spirit_Liquor // 1000
	sellitem Pumpkin_Pie_ // 1000
	sellitem Sunset_On_The_Rock // 1000
	sellitem Morning_Dew // 20
	sellitem Grilled_Rice_Cake // 20
	sellitem 25231 // 1000
	sellitem 25232 // 1000
	sellitem 25233 // 1000
	sellitem Well_Ripened_Berry // 20
	sellitem 11616 // 1400
```

Reply:
25377 = Luxurious Pet Food
25231 = Suspicious Bottle
25232 = Cheap Lubricant
25233 = Cotton Wads
11616 = Delicious Pork

These are from new Pets, which aren't present in rA db